### PR TITLE
Update all python instrumentation docs

### PR DESCRIPTION
**Pull request by srikanthccv**

### CODE REVIEW

File: docs/instrumentation/django.md [hunk 1]
```diff
@@ -65,14 +65,17 @@ You can use OpenTelemetry to send your traces directly to SigNoz. OpenTelemetry
...
-    💡 The `opentelemetry-exporter-otlp` is a convenient way to install all supported OpenTelemetry exporters. Currently it installs:
+    💡 The `opentelemetry-exporter-otlp` is a convenience wrapper package to install all OTLP exporters. Currently, it installs:
+    
     - opentelemetry-exporter-otlp-proto-http
     - opentelemetry-exporter-otlp-proto-grpc
+
+    - (soon) opentelemetry-exporter-otlp-json-http
     
-    We recommend using the http exporter for sending data to SigNoz.
+    The `opentelemetry-exporter-otlp-proto-grpc` package installs the gRPC exporter which depends on the `grpcio` package. The installation of `grpcio` may fail on some platforms for various reasons. If you run into such issues, or you don't want to use gRPC, you can install the HTTP exporter instead by installing the `opentelemetry-exporter-otlp-proto-http` package. You need to set the `OTEL_EXPORTER_OTLP_PROTOCOL` environment variable to `http/protobuf` to use the HTTP exporter.
```
Comment (ankit01-oss) [inline docs/instrumentation/django.md:78]: Will the user know where to set `OTEL_EXPORTER_OYLP_PROTOCOL`?

File: docs/instrumentation/django.md [hunk 2]
```diff
@@ -65,14 +65,17 @@ You can use OpenTelemetry to send your traces directly to SigNoz. OpenTelemetry
 
     `opentelemetry-distro` - The distro provides a mechanism to automatically configure some of the more common options for users. It helps to get started with OpenTelemetry auto-instrumentation quickly. 
     
-    `opentelemetry-exporter-otlp` - This library provides a convenient way to install all supported OpenTelemetry Collector Exporters. You will need an exporter to send the data to SigNoz.
-    
+    `opentelemetry-exporter-otlp` - This library provides a way to install all OTLP exporters. You will need an exporter to send the data to SigNoz.
+
     :::note
-    💡 The `opentelemetry-exporter-otlp` is a convenient way to install all supported OpenTelemetry exporters. Currently it installs:
+    💡 The `opentelemetry-exporter-otlp` is a convenience wrapper package to install all OTLP exporters. Currently, it installs:
```
Comment (ankit01-oss) [inline docs/instrumentation/django.md]:
```suggestion
    💡 The `opentelemetry-exporter-otlp` is a convenient wrapper package to install all OTLP exporters. Currently, it installs:
```

**Review by ankit01-oss — COMMENTED** [review]
A couple of comments otherwise LGTM @srikanthccv .

File: docs/instrumentation/django.md (hunk 1 above)
Comment (srikanthccv) [reply docs/instrumentation/django.md:78]: Did mean where to set or what to set? If it is former, it goes in the same place where other `OTEL_*` go. If it is the latter, two values work 1. `grpc` 2. `http/protobuf`. I have added them in the example run command.

Comment (ankit01-oss) [reply docs/instrumentation/django.md:78]: Yeah, I noticed in the sample app instrumentation that we set it in the final run command.

**Comment by ankit01-oss** [comment]
Looks good to me to merge @srikanthccv. I will update the blogs accordingly.

**Pull request merged**